modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -19,37 +19,38 @@ To specify the pod labels, you use one or more `matchLabels` key-value pairs. If
 .Example `ClusterLogForwarder` CR YAML file
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
+  managementState: Managed
   pipelines:
-    - inputRefs: [ myAppLogData ] <3>
-      outputRefs: [ default ] <4>
-  inputs: <5>
+    - inputRefs: [ myAppLogData ] <1>
+      outputRefs: [ default ] <2>
```

Comment:
use something other then "default" since that has context in 5.9 we no longer support.  user's can use default themselves but let's not make them think it means something to us